tests/functional/amdModuleCircular.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+	<head lang="en">
+	<meta charset="UTF-8">
+	<title>AMD Loading Test</title>
+</head>
+
+<body>
+	<script src="../../loader.js"></script>
```

Comment:
The context of this was the non-indentation of the code below.  I think we could just indent the code in these files one level beneath the script tag's indentation and it'd be fine.

Also, you seem to have some lopsided indentation in `head` in this file.